bigquery/google/cloud/bigquery/magics.py

```diff
@@ -236,6 +237,32 @@ def use_bqstorage_api(self):
     def use_bqstorage_api(self, value):
         self._use_bqstorage_api = value
 
+    @property
+    def maximum_bytes_billed(self):
+        """int: Maximum bytes to be billed for this job or :data:`None` if not set.
+
+        Example:
+            Manually setting the context maximum_bytes_billed:
+
+            >>> from google.cloud.bigquery import magics
+            >>> magics.context.maximum_bytes_billed = '123'
+
+        Raises:
+            ValueError: If the parameters are invalid.
+        """
+        return self._maximum_bytes_billed
+
+    @maximum_bytes_billed.setter
+    def maximum_bytes_billed(self, value):
+        try:
+            value = int(value)
+            self._maximum_bytes_billed = value
```

Comment:
Why do we need `self._maximum_bytes_billed`?  Can't we just use `job_config.maximum_bytes_billed`?

Reply:
Why is `job_config` even in this function?

One way of implementing this feature would be to create a `self._job_config` to hold the default job config (used when constructing the BigQuery client), but that doesn't appear to be what's going on here.